//
// Academic License - for use in teaching, academic research, and meeting
// course requirements at degree granting institutions only.  Not for
// government, commercial, or other organizational use.
//
// acdefg_T_P.h
//
// Code generation for function 'acdefg_T_P'
//

#ifndef ACDEFG_T_P_H
#define ACDEFG_T_P_H

// Include files
#include "rtwtypes.h"
#include <cstddef>
#include <cstdlib>

// Function Declarations
void acdefg_T_P(double P_init, double V_init, double A_init, double P_wayp,
                double V_max, double V_min, double A_min, double J_max,
                double J_min, double T, creal_T t[21]);

#endif
// End of code generation (acdefg_T_P.h)
